kvmagent/kvmagent/plugins/physical_nic_monitor.py

```
@@ -224,7 +225,16 @@ def physical_nic_monitor(self):
-                if time_lock_now != self.time_lock:
-                    break
-                self.physical_nic_monitor_get(ip)
+            try:
+                while True:
+                    if time_lock_now != self.time_lock:
+                        break
+                    try:
+                        readable, _, _ = select.select([ip.fileno()], [], [], 5.0)
+                        if readable:
+                            self.physical_nic_monitor_get(ip)
+                    except Exception as e:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)